app/searchApi.rb

```diff
@@ -298,6 +302,7 @@ def normalizeResponse(response)
 #  "info_count"=>12,        <-- different
 #  "infoResults"=> ...      <-- different
 def searchByType(params, facetTypes=$allFacets.keys, search_type)
+  params.delete("q") if isQueryEmpty(params)
```

Comment:
Whoa, super tricky. Is this the magic that searches for everything if no query is specified. Nice.

Might be worth a comment here when you get around to it.